hydromt_wflow/utils.py

```diff
@@ -142,6 +149,10 @@ def read_csv_results(
                         mask = maps[
                             f"{config['input'].get('reservoir_location__count')}"
                         ].copy()
+                    elif "lake" in col["parameter"]:
+                        mask = maps[
+                            f"{config['input'].get('lake_location__count')}"
+                        ].copy()
```

Comment:
Lakes are removed in Wflow.jl version 1. I guess you had to merge main with this branch at some points? Between this and the change for the conversion functions, seems it comes from conflict with the reservoir/lakes branch I made.
```suggestion

```